Library/Homebrew/cask/spec/cask/artifact/zap_spec.rb

```diff
@@ -8,12 +8,24 @@
     Hbc::Artifact::Zap.new(cask, command: Hbc::FakeSystemCommand)
   }
 
+  let(:test_home_path) { TEST_FIXTURE_DIR }
+  let(:original_home_path) { ENV['HOME'] }
+  let(:absolute_path) { TEST_FIXTURE_DIR/'cask'/'absolute_path' }
+  let(:path_with_tilde) { TEST_FIXTURE_DIR/'cask'/'path_with_tilde' }
+  let(:glob_path1) { TEST_FIXTURE_DIR/'cask'/'glob_path1' }
+  let(:glob_path2) { TEST_FIXTURE_DIR/'cask'/'glob_path2' }
+
   before(:each) do
+    ENV['HOME'] = test_home_path
     shutup do
       InstallHelper.install_without_artifacts(cask)
     end
   end
 
+  after(:each) do
+    ENV['HOME'] = original_home_path
```

Comment:
No need to restore this, the `spec_helper.rb` already makes sure to restore the `ENV` after each test.